pypesto/prediction/prediction.py

```diff
@@ -202,6 +209,14 @@ def write_to_h5(self,
                     f.create_dataset(f'{i_cond}/{OUTPUT_SENSI}',
                                      data=cond.output_sensi)
 
+    def write_to_petab_file(self,
+                            output_file: str,
+                            measurement_df: pd.DataFrame):
+        """
+        This function writes prediction results to a petab simulation file,
+        which is shaped similarly to the measurement DataFrame, which is passed
+        """
```

Comment:
Uuuh, I would argue this function doesn't really do anything ;). In case this is intended as abstract method, please raise `NotImplementedError()`.

Reply:
Nope, that's a "I started implementing this and then ended up doing it another way" :joy: 
Good you found it. Yeah, I may rework this one. Currently "PEtab" export only works for predictions without post-processing. This function could allow to do this also for post-processed settings... Let's see. Either I'll remove or I'll finish it...